library/Fission/Internal/Orphanage/OpenUnion.hs

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Fission.Internal.Orphanage.OpenUnion () where
+
+import RIO
+import Data.WorldPeace
+
+instance Display (OpenUnion '[]) where
+  -- As a base case for the type-level recursion only. Essentially @Void@.
+  display = absurdUnion
+
+instance (Display a, Display (OpenUnion as)) => Display (OpenUnion (a ': as)) where
```

Comment:
Is this to easily render OpenUnion errors to text?

Reply:
Yup! The power of type classes 💪